geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStore.scala

```diff
@@ -357,6 +357,7 @@ class AccumuloDataStore(val connector: Connector,
       val deleter = connector.createBatchDeleter(name, auths, numThreads, defaultBWConfig)
       table.deleteFeaturesForType(sft, deleter)
       deleter.close()
+      if (table == Z3Table && tableOps.exists(name)) tableOps.delete(name)
```

Comment:
you could actually put the deleter code in the else statement for this if, as we don't need to bother deleting all the rows if we're dropping the table. Deleting the rows is fairly slow, so it would be good to avoid.